pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<artifactId>opensrp-server-core</artifactId>
 	<packaging>jar</packaging>
-	<version>2.5.24-SNAPSHOT</version>
+	<version>2.6.0-SNAPSHOT</version>
```

Comment:
Very little has changed to warrant this bump in version. We may need to adopt 4 digits in versioning